ui/helpers/utils/conversions.util.js

```diff
@@ -80,6 +80,7 @@ export function getWeiHexFromDecimalValue({
   conversionRate,
   fromDenomination,
   invertConversionRate,
+  numberOfDecimals,
```

Comment:
It seems that this new `numberOfDecimals` parameter is not used anywhere

Reply:
Yes, needs to be removed didn't end up needing it!